### ISSUE TITLE
Extend image size utility to follow redirects

### ISSUE THREAD

**Issue by kirrg001** [TryGhost/Ghost#8589]
Right now our [image size utility](https://github.com/TryGhost/Ghost/blob/lts/core/server/utils/image-size-from-url.js) does not follow redirects.

#### Here is a case
You run your Ghost frontend with http.
Your admin panel is https.
The admin assets (e.g. `/ghost/img/ghosticon.jpg`) are served depending on the Ghost frontend protocol, [see](https://github.com/TryGhost/Ghost/blob/lts/core/server/middleware/index.js#L143). So Ghost does not force ssl for admin assets!
Imagine a web server (e.g. nginx) redirects all admin traffic to https. The image size utility is then not able to detect the image size of the admin assets.

#### At the end it results in:
```
ERROR: Unknown Request error. 
 http://XXXX.ghost.io/ghost/img/ghosticon.jpg 
 
 Error: Unknown Request error.
    at IncomingMessage.<anonymous> (/home/ghost/core/server/utils/image-size-from-url.js:79:31)
```

- [x]  follow redirects by hacking it into the image size utility

**Comment by AileenCGN** [comment]
@kirrg001 Does it make sense in your opinion to use `got` in LTS as well?

**Comment by AileenCGN** [comment]
This is fixed in master already. LTS PR #8953 is updated and will finally fix this.

**Comment by AileenCGN** [comment]
Closed with b1d829d9b6813f753442ba5b3e2bcfbb16ebec63
